test/carbon/Pairs.ts

```diff
@@ -1,19 +1,27 @@
 import Contracts, { CarbonController, TestERC20Burnable } from '../../components/Contracts';
 import { ZERO_ADDRESS } from '../../utils/Constants';
+import { Roles } from '../../utils/Roles';
 import { createSystem, createTestToken } from '../helpers/Factory';
 import { shouldHaveGap } from '../helpers/Proxy';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
 import { expect } from 'chai';
+import { ethers } from 'hardhat';
 
 const sortTokens = (token0: string, token1: string): string[] => {
     return token0 < token1 ? [token0, token1] : [token1, token0];
 };
 
 describe('Pairs', () => {
     let carbonController: CarbonController;
+    let emergencyStopper: SignerWithAddress;
     let token0: TestERC20Burnable;
     let token1: TestERC20Burnable;
     let pair: any;
 
+    before(async () => {
+        [, , emergencyStopper] = await ethers.getSigners();
```

Comment:
Minor: can probably get rid of one of the commas here

Reply:
well we need the third signer only, and we don't need the first two, so if I remove a comma, it's going be the incorrect signer.